src/explore-education-statistics-frontend/src/modules/find-statistics/ReleaseExploreDataPage.tsx

```diff
@@ -0,0 +1,460 @@
+    supportingFiles,
+    featuredTables,
+  } = dataContent;
+
+  const hasSupportingFiles = supportingFiles.length > 0;
+  const hasFeaturedTables = featuredTables.length > 0;
+  const hasDataDashboards = dataDashboards && dataDashboards.length > 0;
+
+  const { isMedia: isMobileMedia } = useMobileMedia();
+
+  const [showAllDataSetDetails, toggleAllDataSetDetails] = useToggle(false);
+
+  const featuredTablesContent = (
+    <ReleaseDataList
+      heading={`${featuredTables.length} featured table${
+        featuredTables.length > 1 ? 's' : ''
+      }`}
+    >
+      {featuredTables.map(featuredTable => (
+        <ReleaseDataListItem
+          key={featuredTable.featuredTableId}
+          title={featuredTable.title}
+          description={featuredTable.summary}
+          actions={
+            <Link
+              to={`/data-tables/fast-track/${featuredTable.dataBlockParentId}`}
+            >
+              View, edit or download{' '}
+              <VisuallyHidden>{featuredTable.title}</VisuallyHidden>
+            </Link>
+          }
+        />
+      ))}
+    </ReleaseDataList>
+  );
+
+  const dataSetsContent = (
+    <ReleaseDataList
+      heading={`${dataSets.length} data set${
+        dataSets.length > 1 ? 's' : ''
+      } available for download`}
+      actions={
+        <Link
+          to={`${process.env.CONTENT_API_BASE_URL}/releases/${releaseVersionSummary.id}/files?fromPage=ReleaseDownloads`}
+          onClick={() => {
+            logEvent({
+              category: 'Downloads',
+              action: `Release page all files, Release: ${releaseVersionSummary.title}, File: All files`,
+            });
+          }}
+          className="govuk-!-font-weight-bold"
+          unvisited
+        >
+          Download all (ZIP)
+        </Link>
+      }
+      toggle={
+        <AccordionToggleButton
+          expanded={showAllDataSetDetails}
+          label={
+            showAllDataSetDetails
+              ? 'Hide all details'
+              : 'Show expanded view for all data sets'
+          }
+          onClick={() => {
+            toggleAllDataSetDetails();
+            logEvent({
+              category: 'Data catalogue',
+              action: 'All data set details toggled',
+            });
+          }}
+        />
+      }
+    >
+      {dataSets.map(dataset => (
+        <ReleaseDataListItem
+          key={dataset.fileId}
+          title={dataset.title}
+          description={dataset.summary}
+          metaInfo={dataset.meta.geographicLevels.join(', ')}
+          actions={
+            <>
+              <Link
+                to={`/data-tables/${publicationSummary.slug}/${releaseVersionSummary.slug}?subjectId=${dataset.subjectId}`}
+              >
+                Create table{' '}
+                <VisuallyHidden>using {dataset.title}</VisuallyHidden>
+              </Link>
+              <ButtonText
+                onClick={async () => {
+                  await downloadService.downloadZip(
```

Comment:
I think this is probably a back end issue - the file name is the same (the publication & release name) whether you're downloading all data or individual data sets. I'd expect it to be the data set name for the individual downloads.